test/WebSites/ModelBindingWebSite/Models/Employee.cs

```diff
@@ -9,5 +11,14 @@ public class Employee : Person
         public string Department { get; set; }
 
         public string Location { get; set; }
+
+        [FromQuery(Name = "EmployeeId")]
+        public int Id { get; set; }
+
+        [FromRoute(Name = "EmployeeTaxId")]
+        public int TaxId { get; set; }
+
+        [FromForm(Name = "EmployeeSSN")]
+        public string SSN { get; set; }
```

Comment:
There is already a ```[ModelBinder(Name="")]``` attribute present. The intention here is to test From* attributes not so much as to showcase this as a sample.